app/server/views/app.hbs

```diff
@@ -3,7 +3,8 @@
     <head>
         <meta charset="utf-8">
         <title>{{title}}</title>
-        <link rel="stylesheet" href="{{baseUrl}}/css/wodin.css"
+        <link rel="stylesheet" href="{{baseUrl}}/css/wodin.css"/>
+        <script src="https://kit.fontawesome.com/8e829cc319.js" crossorigin="anonymous"></script>
```

Comment:
It actually doesn't work well with scss! I remembered why I didn't use npm or a ttf file. For some reason webpack and scss have this bug where webpack can't see some relative imports in scss files. I am not sure why this is but I remember trying to get it working but failing to do so. Monaco needs a class to assign to the glyph so I initially tried with images and src url but that didn't work either because of the relative import problem. Perhaps we can use an scss files/css file for font awesome that doesn't have any relative imports? Should I try that?

Reply:
I think that would be better than having a raw script link in the template - good to keep all the front end imports together as much as possible I think. Would be good to get to the bottom of the webpack/scss issue though - did you find if anyone else was having that issue or if there was an open issue for it?